projects/control-service/projects/helm_charts/pipelines-control-service/templates/deployment.yaml

```diff
@@ -80,6 +80,8 @@ spec:
               value: "{{ .Values.deploymentDockerRegistryUsername }}"
             - name: DOCKER_REGISTRY_PASSWORD
               value: "{{ .Values.deploymentDockerRegistryPassword }}"
+            - name: DOCKER_REGISTRY_SECRET
```

Comment:
shouldn't we have the same check here as at Secret?

Reply:
I added the check, plus checking if deploymentDockerRegistryType is generic.